just_audio/example/lib/main.dart

```diff
@@ -71,7 +71,7 @@ class _MyAppState extends State<MyApp> {
   }
 
   Future<void> _init() async {
-    final session = await AudioSession.instance;
+    final session = await AudioSession.isInitialized;
```

Comment:
I do not understand this pull request as I haven't provided any getter in audio_session called `isInitialized`.

Reply:
Ohh, Yes you are correct. It was supposed to be on different repository, but by mistake I did on this repo.

Sorry

Reply:
Ah, OK :-) Mystery solved.

Reply:
:smiley: